Data-Science/README.md

```diff
@@ -0,0 +1,176 @@
+- [CS480/680 -Youtube(Pascal Poupart)](https://youtube.com/playlist?list=PLdAoL1zKcqTW-uzoSVBNEecKHsnug_M0k)
+- [Dive into Deep Learning -Github(d21-ai)](https://github.com/d2l-ai/d2l-en?fbclid=IwAR0QN35b-NHHWq_zKISA1cbI063aRqqoKqR_0e3cpnT5h58GkcNbCIJs3iw)
+- [Introduction to Deep Learning -GFG(geeksforgeeks)](https://www.geeksforgeeks.org/introduction-deep-learning/)
+
+### PyTorch
+
+- [PyTorch -Youtube(Dr.Data Science)](https://www.youtube.com/watch?v=vVQs4h6HUvA&list=PLLeO8f6PhlKb_FAC7qxOBtxT9-8EPDAqk)
+- [PyTorch -Youtube(UC Berkeley) - Lec3 (The 5 parts)](https://m.youtube.com/watch?v=AOypIa_8RXg&list=PL_iWQOsE6TfXxKgI1GgyV1B_Xa0DxE5eH&index=11)
+- [Pytorch Tutorial -Youtube(Aladdin Persson)](https://www.youtube.com/watch?v=2S1dgHpqCdk&list=PLhhyoLH6IjfxeoooqP9rhU3HJIAVAJ3Vz)
+- [PyTorch Course (2022) - Youtube(Mr. p Solver)](https://www.youtube.com/watch?v=v43SlgBcZ5Y&list=PLkdGijFCNuVk9fO1IMfdV1Igob0FUHhkB)
+- [Deep Learning With Pytorch -GoogleDrive(Eli Stevens, Luca Antiga)](https://drive.google.com/file/d/1-KG_ufeg7zw2iLgG5RrJSFpyonLwulgF/view?usp=sharing)
+
+### Natural Language Processing (NLP)
+
+- [Introduction to Natural Language Processing in Python -datacamp](https://www.datacamp.com/courses/natural-language-processing-fundamentals-in-python)
+- [NLP tutorial python playlist -Youtube(codebasics)](https://youtube.com/playlist?list=PLeo1K3hjS3uuvuAXhYjV2lMEShq2UYSwX)
+- [NLP with spaCy & python -Youtube(freeCodeCamp.org)](https://youtu.be/dIUTsFT2MeQ)
+- [NLP introduction documentation -GFG(geeksforgeeks)](https://www.geeksforgeeks.org/introduction-to-natural-language-processing/)
+
+### MODEL Deployment
+
+- [Flask tutorial -towardsdatascience](https://towardsdatascience.com/deploying-a-deep-learning-model-using-flask-3ec166ef59fb)
+- [Flask tutorial -GFG(geeksforgeeks)](https://www.geeksforgeeks.org/flask-creating-first-simple-application/)
+- [How to Deploy a Machine Learning Model to Google Cloud -Youtube(Daniel Bourke)](https://www.youtube.com/watch?v=fw6NMQrYc6w)
+- [GCP tutorial -GFG(geeksforgeeks)](https://www.geeksforgeeks.org/what-is-google-cloud-platform-gcp/)
+- [Deploy Models with TensorFlow Serving and Flask -Coursera(Amit Yadav)](https://www.coursera.org/projects/deploy-models-tensorflow-serving-flask)
+
+### Sample PROJECT Ideas
+
+- [End to end Project](https://www.youtube.com/playlist?list=PLatl6hdtJ0RnbkReSAuel6PeCPO155FpG)
+- [Machine Learning Projects](https://www.youtube.com/watch?v=fiz1ORTBGpY&list=PLfFghEzKVmjvuSA67LszN1dZ-Dd_pkus6)
+- [End To End ML Project With Dockers,Github Actions And Deployment](https://www.youtube.com/watch?v=MJ1vWb1rGwM)
+- [Real-World Python Machine Learning Tutorial w/ Scikit Learn](https://youtu.be/M9Itm95JzL0)
+
+### BOOKS
+
+- [Drive Link for books](<(https://drive.google.com/drive/folders/1iW7IPrVUqsHumgXUMH_rgeBLpJjRDCmJ?usp=sharing)>)
+- [Probabilistic Machine Learning: An Introduction](https://probml.github.io/pml-book/book1.html)
```

Comment:
This is more of a Machine Learning Book and not a data science.
This would make much more sense if it's in the Machine Learning Roadmap and not here.

Reply:
- removed it